src/components/PostComment.jsx

```diff
@@ -0,0 +1,40 @@
+import {useState, useContext} from "react"
+import "./postComment.css"
+import {UserContext} from "../contexts/User"
+import { Login } from "../contexts/Login"
+import { postComment } from "../api"
+function PostComment({comments, setComments, article_id}){
+    const [comment, setComment] = useState("")
+    const {user, setUser} = useContext(UserContext)
+    const {setLogin} = useContext(Login)
+    const loginPop = document.getElementsByClassName("login-form")
+    const submitHandle = (e) => {
```

Comment:
This function won't prevent a user from clicking submit multiple times on a slow connection where the first request hasn't been completed yet. Stop the user from being able to submit multiple times